common/src/main/java/org/jboss/hal/testsuite/fragment/shared/table/ResourceTableRowFragment.java

```diff
@@ -49,4 +51,8 @@ public void view() {
 
         Graphene.waitGui().withTimeout(1500, TimeUnit.MILLISECONDS);
     }
+
+    public boolean isSelected() {
```

Comment:
Javadoc, to make clear what is selected, whole table, row in table or cell in table,...